playwright/network.py

```diff
@@ -173,3 +173,18 @@ def request(self) -> Request:
     @property
     def frame(self) -> "Frame":
         return self.request.frame
+
+
+def serialize_headers(headers: Dict[str, str]) -> List[Header]:
+    headers_list = list()
+    for name in headers:
+        header: Header = dict(name=name, value=headers[name])
+        headers_list.append(header)
+    return headers_list
```

Comment:
that would replace 5 lines: `[{"name": name, "value": value} for name, value in x.items()]`

But probably also a personal preference.

Reply:
Done. That's what happens when one doesn't know Python.